nibabel/externals/__init__.py

```diff
@@ -1,5 +1,5 @@
 # init for externals package
-try:
-    from collections import OrderedDict
-except ImportError:  # < Python 2.7
-    from .ordereddict import OrderedDict
+from collections import OrderedDict
+
+from ..deprecated import ModuleProxy as _ModuleProxy
+six = _ModuleProxy('nibabel.externals.six')
```

Comment:
As nibabel directly uses six everywhere now, this is just here to support 3rd party code that may still be using nibabels.external.six?